omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeController2_0Test.java

```diff
@@ -95,4 +98,13 @@ public void shouldVoidAttribute() throws Exception {
 		Assert.assertTrue(conceptAttribute.isVoided());
 		Assert.assertEquals("unit test", conceptAttribute.getVoidReason());
 	}
+	
+	@Test
+	public void shouldGetAConceptAttributeByUuid() throws Exception {
+	MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
```

Comment:
Can you do something about this indention?

Reply:
> Can you do something about this indention?

@dkayiwa I guess have responded appropriately!